src/Php56/bootstrap.php

```diff
@@ -21,4 +21,16 @@ function hash_equals($knownString, $userInput) { return p\Php56::hash_equals($kn
 
         function ldap_escape($subject, $ignore = '', $flags = 0) { return p\Php56::ldap_escape($subject, $ignore, $flags); }
     }
+
+    // Missing functions in PHP 5.5.9 - affects 32 bit builds of Ubuntu 14.04LTS
+    // See https://bugs.launchpad.net/ubuntu/+source/php5/+bug/1315888
+    if (!function_exists('gzopen') && function_exists('gzopen64')) {
+        function gzopen($filename , $mode = 'r', $use_include_path = 0 ) { return gzopen64($filename, $mode, $use_include_path); }
+    }
+    if (!function_exists('gzseek') && function_exists('gzseek64')) {
+        function gzseek($zp , $offset, $whence = SEEK_SET) { return gzseek64($zp, $offset, $whence); }
+    }
+    if (!function_exists('gztell') && function_exists('gztell64')) {
+        function gztell($zp) { return gztell64($zp); }
```

Comment:
i'm ok with not duplicating the function as static method in the Php56 class (reduces boiler plates)